profiler/data/prfiler_excluding.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+
+        <!-- exclude_fnames de pstats_print2list -->
+        <record id="default_exclude_fnames_pstats_print2list" model="ir.config_parameter">
+            <field name="key">exclude_fnames</field>
+            <field name="value">['/.repo_requirements', '/root/odoo-10.0', '/usr/', '>']</field>
```

Comment:
In order to avoid a `eval()` better use items separated by commas
I mean,
`<field name="value">/.repo_requirements,/root/odoo-10.0,/usr/</field>`

And use it with `value.split(',')`